README.md

```diff
@@ -95,9 +95,15 @@ manually load the driver via `modprobe asix`.
 
 On most platforms, the default behavior of Nerves is to hang if something goes wrong.
 This is good for debugging since rebooting makes it easy to lose console
-messages or it might hide the issue completely. Hanging on the EV3 forces you
-to remove and reinsert the batteries since there's no reboot button. The default
-behavior has been changed to power down instead.
+messages or it might hide the issue completely. On the EV3 hanging
+requires a slightly more complex restarting process.
+
+  1. Hold down the Back, center, and left buttons on the EV3 Brick .
```

Comment:
Holding down the left button doesn't actually do anything.

Just press and hold the back and center buttons until the power goes off, then release both buttons at the same time.